src/services/cache.ts

```diff
@@ -55,7 +62,7 @@ export class Cache {
   }
 
   async expire(key: string, ttlSeconds: number) {
-    await this.local.expire(key, ttlSeconds)
+    await this.local.expire(key, this.getLocalTTlL(ttlSeconds))
```

Comment:
there's an extra 1 in this function name

Reply:
l you mean haha but you're right, on it